maintenance/wikia/createUser.php

```diff
@@ -35,7 +35,7 @@
 if ( !is_object( $user ) ) {
 	echo( "invalid username.\n" );
 	die( 1 );
-} elseif ( 0 != $user->idForName() ) {
+} elseif ( 0 != $user->getId() ) {
 	echo( "account exists.\n" );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the parentheses from this "echo" call. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS2041)

[project-id]: wikia:app